compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -429,7 +429,6 @@ E0700: include_str!("./error_codes/E0700.md"),
 E0701: include_str!("./error_codes/E0701.md"),
 E0703: include_str!("./error_codes/E0703.md"),
 E0704: include_str!("./error_codes/E0704.md"),
-E0705: include_str!("./error_codes/E0705.md"),
```

Comment:
Please put it back and update the explanation to say it's not emitted anymore (CI will fail in any case explaining this too :wink:).

Reply:
Yep. I saw others were "removed" and didn't bother to read all the comments.